### PR TITLE
Android Implementation for Native Mixpanel

### PR DESCRIPTION
**Pull request by psovit**
#2

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -2,15 +2,15 @@ group 'com.example.native_mixpanel'
...
+    ext.kotlin_version = '1.3.21'
     repositories {
         google()
         jcenter()
     }
 
     dependencies {
         classpath 'com.android.tools.build:gradle:3.2.1'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
```
Comment (surenyonjan) [inline android/build.gradle]: @psovit extra space at the end

File: android/src/main/kotlin/com/example/native_mixpanel/NativeMixpanelPlugin.kt [hunk 2]
```diff
@@ -18,8 +27,14 @@ class NativeMixpanelPlugin: MethodCallHandler {
   override fun onMethodCall(call: MethodCall, result: Result) {
     if (call.method == "getPlatformVersion") {
```
Comment (surenyonjan) [inline android/src/main/kotlin/com/example/native_mixpanel/NativeMixpanelPlugin.kt]: @psovit can we remove this call to `getPlatformVersion` ?

File: android/build.gradle (hunk 1 above)
Comment (psovit) [reply android/build.gradle]: fixed by removing the extra space

File: android/src/main/kotlin/com/example/native_mixpanel/NativeMixpanelPlugin.kt (hunk 2 above)
Comment (psovit) [reply android/src/main/kotlin/com/example/native_mixpanel/NativeMixpanelPlugin.kt]: fixed by removing the call to `getPlatformVersion`

**Pull request merged**